reframe/frontend/cli.py

```diff
@@ -360,6 +360,7 @@ def main():
 
     logging.getlogger().colorize = site_config.get('general/0/colorize')
     printer.colorize = site_config.get('general/0/colorize')
+    printer.inc_verbosity(site_config.get('general/verbose'))
```

Comment:
```suggestion
    printer.inc_verbosity(site_config.get('general/0/verbose'))
```

Reply:
We will need to to the same [here](https://github.com/eth-cscs/reframe/blob/master/reframe/frontend/cli.py#L336).